### ISSUE TITLE
watchOS notifications don't work when in jailbroken state

### ISSUE THREAD

**Issue by rangeroverct** [checkra1n/BugTracker#75]
**What device + iOS version are you on?**
iPhone 7 GSM (iPhone9,3) iOS 13.2.2 (17B102)
Apple Watch Series 4 (watch4,2) watchOS 6.1 (17S84)

checkra1n 0.9 beta on late 2014 Mac Mini running 10.15 catalina

**What are the steps to reproduce the issue?**
1. Have a paired Apple Watch
2. Jailbreak with checkra1n
3. Have someone send you a message 
...

**What do you expect, and what is happening instead?**
The notification just appears on the iPhone, and doesn't appear at all on the watch.
Works in non-jailbroken state.
Tried re-pairing watch and installing [[Always Forward watchOS notifications]]

**Any other info, error logs, screenshots, ...?**
None

**Comment by nullpixel** [comment]
Do you have any tweaks installed?

**Comment by TheNoim** [comment]
Ok, it seems like I have the same issue. 

iOS: 13.2.2 iPhone9,4
watchOS: 6.1 (17S84) Series 3 LTE model without LTE :/
Checkra1n: 0.9.1

Btw: The watch app also seems like it can not load the state of the watch. At least my app list and watch faces doesn't load. 

I can not find any crash report related to this. 

Current list of installed tweaks:

```
applist
apt
apt-key
apt1.4
apt7-key
base
bash
berkeleydb
bzip2
ca-certificates
cokepokes.jp.ashikase.libpackageinfo
com.bingner.snappy
com.cokepokes.libnotifications
com.dave1482.powerapp
com.exile90.icleanerpro
com.foxfort.amazonite
com.foxfort.libfoxfortsplash
com.foxfort.libfoxfortutils
com.imkpatil.floatingdockplus
com.linusyang.localeutf8
com.muirey03.cr4shed
com.randomblock1.select1337
com.rango.trueshuffle
com.rpetrich.rocketbootstrap
com.sassoty.pulltorespring
com.saurik.substrate.safemode
com.spark.libsparkapplist
com.spos
com.tigisoftware.filza
coreutils
coreutils-bin
crash-reporter
cydia
cydia-lproj
darwintools
debianutils
diffutils
diskdev-cmds
dpkg
essential
file
findutils
firmware-sbin
flex3beta
gcrypt
gettext
gnupg
gnutls
grep
gzip
jp.ashikase.libcrashreport
jp.ashikase.libpackageinfo
jp.ashikase.libsymbolicate
jp.ashikase.techsupport
ldid
libapt
libapt-pkg5.0
libassuan
libc-ares2
libgmp10
libgpg-error
libidn2
libksba
libnghttp2-14
libplist
libssl1.0
libssl1.1.1
libtasn1
libunistring
lz4
lzma
me.jjolano.shadow
mobilesubstrate
ncurses
ncurses5-libs
nettle
nodejs
npth
openssh
org.thebigboss.libcolorpicker
org.thebigboss.repo.icons
p11-kit
p7zip
preferenceloader
profile.d
readline
sed
shell-cmds
system-cmds
tar
uikittools
unrar
unzip
wget
ws.hbang.common
ws.hbang.newterm2
xyz.willy.zebra
xz
zip
```

**Comment by vanwijkdave** [comment]
I have an S4 apple watch and an iPX on iOS 13.1 and it works fine for me

**Comment by TheNoim** [comment]
Maybe then it is an ios 13.2.2 issue

**Comment by PatrikTheDev** [comment]
I have the same issue on 13.3, it works once I reboot. It also happens in safe mode, so it’s not caused by a tweak. watchOS 6.1.1

**Comment by PatrikTheDev** [comment]
Link to my tweet about the issue: [Link](https://twitter.com/patrikthedev/status/1193614725181059080?s=21)  (tagged @nullpixel there)

**Comment by TheNoim** [comment]
I hope they will look into this issue. However, this is a beta and I expected issues like this.

**Comment by PatrikTheDev** [comment]
Also if it helps them, the app doesn’t load anything for me too.

**Comment by nullpixel** [comment]
Thanks for the feedback, I’ll take a look at this when I have time tomorrow and try and reproduce it myself.

**Comment by yur1xpp** [comment]
I have all the exact same issue

**Comment by StefanNienhuis** [comment]
Also happens on iOS 13.2 paired with a Series 1

**Comment by TheNoim** [comment]
Did anybody find a workaround? It is a bit annoying to not get your notifications on your watch.

**Comment by PatrikTheDev** [comment]
The weird, it fixed itself for me...

**Comment by TheNoim** [comment]
Maybe it is a issue with the daemon. I would like to try to restart it. But I don‘t know what its name is.

**Comment by yur1xpp** [comment]
Also @nullpixel, the watch keep resyncing every time you boot into non-jailbroken state from a jailbroken state, each sync takes roughly 20 minutes. It happen when you open the watch app, then the screen like the screenshot below popped up. Clicking back will reset the watch, which I’m trying to avoid. Hope this helps in debugging. Thanks!

https://imgur.com/a/datUEq6

**Comment by luka001** [comment]
All the same problem here on iOS 13.2.2 iPhone X 10.6 GSM. Chackra1n 0.9.1 on Macbook Air Catalina OS. Even if I turn off wifi watch show like he is not in range of iPhone and cant click find my phone. In watch app there is show like there is no watch. In watch app on phone I can find location of watch. Watch is seria 1, version 5.3.3. 
![AB49F46B-E4F2-4BEB-8EFA-83E0E7EAD170](https://user-images.githubusercontent.com/3748193/68581588-fb7bc280-0478-11ea-9679-57a318d6dd83.png)
![91D765ED-6BEC-496F-A990-315CD87BFA53](https://user-images.githubusercontent.com/3748193/68581589-fc145900-0478-11ea-9f16-afeb6f93cd68.png)
![Uploading 36CAA2B3-8F15-4422-9619-9AB8335BDBD2.jpeg…]()

**Comment by jclegg555** [comment]
I was having the same issue as well. Not sure why but it is working now

**Comment by luka001** [comment]
After restore and fresh reconect to iWatch all work perfect like before jailbreak.

**Comment by nullpixel** [comment]
This is fixed in the build shipping very soon.

**Comment by PatrikTheDev** [comment]
Great! Thanks!

**Comment by nullpixel** [comment]
This issue is fixed in 0.9.2, which is available now at https://checkra.in. Please try it and let me know the results!
